rust/src/pgsql/parser.rs

```diff
@@ -320,6 +320,7 @@ pub enum PgsqlFEMessage {
     SASLResponse(RegularPacket),
     SimpleQuery(RegularPacket),
     Terminate(TerminationMessage),
+    UnknownMessageType(RegularPacket),
```

Comment:
Is there a real difference with `PgsqlBEMessage` ?

Reply:
not in the message datatype as implemented here itself, only to work with returning types that are either `PgsqlFEMessage` or `PgsqlBEMessage`.

Does this answer the question?

Reply:
Thanks for the answer and sorry for this protocol

Reply:
XD